### ISSUE TITLE
[Suggestion]: Daily

### ISSUE THREAD

**Issue by desagas** [someaddons/weathersettings#5]
### Describe the bug you're experiencing

Just a suggestion: currently, the rain still happens randomly, which is Minecraft for you. But, living in or visiting many tropical or sub tropical countries has inspired me, and I hope you:

It would be nice if this mod could make it rain from X time every day to Y time every day, and at no other times, all configurable. Even allowing a list, say rain from 23000ticks to 23500ticks, and 1600ticks to 3000ticks, but a list with the options to add as many as we like. For example, in real life Guatemala, when I stayed, it would thunder every day from 5pm until 6pm and then rain from 6 to 630, give or take a few minutes here or there.

So, a config might look like:

```
# weather: random or forced; default: random
weather: forced

# forced daily weather times, from 00000 to 23999 ticks
# example: ["06000 to 09000"] would rain from 12pm to 3pm in game time.
forced times for rain: ["01600 to 02400", "06000 to 09000", "21000 to 23000"]
forced times for storm: ["02400 to 02550", "09000 to 10500", "23500 to 01000"]
```

### Mod up to date

- [X] Before submitting this issue I updated to the newest version and reproduced it

### Minecraft version

1.21.1

### Modloader version

Neoforge

**Comment by someaddons** [comment]
if you match rain/storm duration and clearDuration to that 1 day cycle then on avg you should already get this
e.g. rain/storm duration to 150 and clearDuration of both to 1050

**Comment by desagas** [comment]
You are correct, but, it is hard to fine tune this if both storms and general rain are both active, and, then, at that point, it would only work if A, one or the other was set to 0 duration (disabled), or B, both were set equally, which might cause conflicts, ie, which one wins? Do you have a config example for what you are suggesting? What you mentioned above would cause rain/storm both at 6am AND 6pm every day, but, you are right, it would kind of work, just not exactly.
